src/telemetry/prometheus/metrics.py

```
@@ -325,6 +325,27 @@ def record_telemetry_event_dropped(self, *, reason: str) -> None:
+        Prometheus/Grafana until an event is actually dropped — you cannot alert
+        on or graph a metric that does not exist yet. Materializing the
+        ``(namespace, reason)`` children at startup keeps the metric present at 0,
+        so a missing series signals a broken scrape rather than "no drops".
+
+        Args:
+            reasons: The reason label values the calling emitter can produce.
+        """
+        for reason in reasons:
+            try:
+                telemetry_events_dropped_counter.labels(reason=reason)
+            except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)